### PR TITLE
add aggregated attestation tracing to logtrace and enable it in Jenkins

### PR DESCRIPTION
**Pull request by tersec**
It'll show something like:
```
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=a26a77f8 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:03Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=a1e88950 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:09Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=8fd33a5c receivers=['log2.txt'] send_stamp=2021-08-06T18:31:09Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=a8334c41 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:09Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=911e35a0 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:15Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=b086e3a6 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:15Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=b7eacef0 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:15Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=b479aac4 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:15Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=ac3cd63b receivers=['log2.txt'] send_stamp=2021-08-06T18:31:15Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=81a5a2d6 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:21Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=abf70410 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:21Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=96ccb249 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:21Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=8975d1f8 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:21Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=b0de2513 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:27Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=842fc452 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:27Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=aa2cf3f0 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:27Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=b2ff4dea receivers=['log2.txt'] send_stamp=2021-08-06T18:31:27Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=aa487334 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:27Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=a37e3308 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:27Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=a219e6b1 receivers=['log2.txt'] send_stamp=2021-08-06T18:31:33Z
INF 2021-08-06 20:31:35.250+02:00 Aggregated attestation was not received    tid=3344062 file=logtrace.nim:600 sender=log3.txt signature=b1c0b0ef receivers=['log2.txt'] send_stamp=2021-08-06T18:31:33Z
INF 2021-08-06 20:31:35.251+02:00 Statistics for sender node                 tid=3344062 file=logtrace.nim:605 sender=log3.txt successful_broadcasts=60 failed_broadcasts=94 total_broadcasts=154
```

There seems to be a consistent phenomenon here of aggregated attestations, which should be broadcast everywhere, not being such reliably.

### CODE REVIEW

File: scripts/launch_local_testnet.sh [hunk 1]
```diff
@@ -69,7 +69,7 @@ CI run: $(basename "$0") --disable-htop -- --verify-finalization
   --base-metrics-port         bootstrap node's metrics server port (default: ${BASE_METRICS_PORT})
   --disable-htop              don't use "htop" to see the nimbus_beacon_node processes
   --disable-vc                don't use validator client binaries for validators (by default validators are split 50/50 between beacon nodes and validator clients)
-  --enable-logtrace           display logtrace asr analysis
+  --enable-logtrace           display logtrace aasr analysis
```
Comment (stefantalpalaru) [inline scripts/launch_local_testnet.sh]: Rather cryptic acronym for "aggregated attestation sent/received" messages.

**Review by stefantalpalaru — APPROVED** [review]

File: scripts/launch_local_testnet.sh (hunk 1 above)
Comment (tersec) [reply scripts/launch_local_testnet.sh]: suggest another?

Comment (tersec) [reply scripts/launch_local_testnet.sh]: logtrace isn't exactly a typical user-facing front-end which benefits from being easily discoverable, either.

Comment (stefantalpalaru) [reply scripts/launch_local_testnet.sh]: There's room to expand the acronym.

Comment (tersec) [reply scripts/launch_local_testnet.sh]: `aggasr` probably makes the parallels more obvious: https://github.com/status-im/nimbus-eth2/pull/2766/commits/61ac3ee78800ce96be8be9b6289c7e9fb2e8e1c9

**Pull request merged**